app/config.py

```diff
@@ -0,0 +1,3 @@
+URL = 'http://dev.hr.excellencetechnologies.in/hr/attendance/API_HR/api.php'
```

Comment:
k. this is wrong.

there will be just based url. rest specific to api put url where code is called.
only one url will be here